pkgs/by-name/az/azahar/package.nix

```diff
@@ -142,6 +143,7 @@ stdenv.mkDerivation (finalAttrs: {
     (cmakeBool "ENABLE_QT_TRANSLATION" enableQtTranslations)
     (cmakeBool "ENABLE_CUBEB" enableCubeb)
     (cmakeBool "USE_DISCORD_PRESENCE" useDiscordRichPresence)
+    (cmakeBool "ENABLE_SSE42" enableSSE42)
```

Comment:
@arthsmn, I think this should use `stdenv.hostPlatform.sse4_2Support` instead of a parameter.